### PR TITLE
Creates clipmenu-alpha.rb

### PR DESCRIPTION
**Pull request by darwingr**
The developer has 3 branches, this is the latest.

### CODE REVIEW

File: Casks/clipmenu-alpha.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'clipmenu-alpha' do
+  version '1.0.0a1'
+  sha256 'ce92a6efa7397d39e0d73bdd5394438ceb1f66543c2e4695969d7acfaad1434e'
+
+  # dropbox.com is the official download host per the vendor homepage
+  url "https://dl.dropbox.com/u/1140644/clipmenu/ClipMenu-#{version}.dmg"
+
```
Comment (vitorgalvao) [inline Casks/clipmenu-alpha.rb]: Please remove this line.

File: Casks/clipmenu-alpha.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  # dropbox.com is the official download host per the vendor homepage
+  url "https://dl.dropbox.com/u/1140644/clipmenu/ClipMenu-#{version}.dmg"
+  
+  name 'ClipMenu'
+  homepage 'http://www.clipmenu.com/'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  
+  app 'ClipMenu.app'
+  depends_on :macos => '>= 10.10'
+
```
Comment (vitorgalvao) [inline Casks/clipmenu-alpha.rb]: Please remove this one as well.

File: Casks/clipmenu-alpha.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  # dropbox.com is the official download host per the vendor homepage
+  url "https://dl.dropbox.com/u/1140644/clipmenu/ClipMenu-#{version}.dmg"
+  
+  name 'ClipMenu'
+  homepage 'http://www.clipmenu.com/'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  
+  app 'ClipMenu.app'
+  depends_on :macos => '>= 10.10'
```
Comment (vitorgalvao) [inline Casks/clipmenu-alpha.rb]: There should be an empty line above this one. In addition, we prefer to call releases by name (`depends_on :macos => '>= :yosemite').

**Comment by vitorgalvao** [comment]
Please point me to the relevant download page so I can verify the download link.

**Comment by darwingr** [comment]
http://www.clipmenu.com/blog/
Released in the latest blog post: November 11, 2014.

Unfortunately the download url pattern uses a dash(-) instead of an underscore(_) like in the main and dev versions.

**Comment by darwingr** [comment]
From what I see it failed again, I think because quotes were left on when I switched 10.10 to yosemite.
The same error is on the CASK_LANGUAGE_REFERENCE.md.

**Comment by vitorgalvao** [comment]
Don’t worry about the Travis failure: there’s currently a problem with it in this repo, failing on correct casks. We’re aware of it.

Everything looks good. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by ndr-qef** [comment]
Thanks.

**Pull request merged**